server.js

```diff
@@ -33,7 +57,8 @@ var allowCrossDomain = function(req, res, next) {
   res.header('Access-Control-Allow-Methods', 'GET');
   res.header('Access-Control-Allow-Headers', 'Content-Type, Authorization, Content-Length, X-Requested-With');
 
-  // intercept OPTIONS method
+  //Question - intercept OPTIONS method - How would the OPTION method ever be called?
+  //Wouldn't it always be a GET?
   if ('OPTIONS' == req.method) {
```

Comment:
@davidlimcheng we actually used this to keep the heroku app up and running by sending a request with `OPTIONS` every 5 mins